cmd/auth/main.go

```diff
@@ -61,54 +61,54 @@ func main() {
 	// Create auth service configurations
 	cfg := config{}
 	if err := env.Parse(&cfg); err != nil {
-		log.Fatalf("failed to load %s configuration : %s", svcName, err.Error())
+		log.Fatalf("failed to load %s configuration : %s", svcName, err)
 	}
 
 	logger, err := logger.New(os.Stdout, cfg.LogLevel)
 	if err != nil {
-		log.Fatalf(err.Error())
+		log.Fatal(fmt.Sprintf("failed to init logger: %s", err))
```

Comment:
Is this `logger` or `log`?

Reply:
This is a native Go `log` because `logger` was not initialized properly. But let me also rename it because we have:
```go
logger, err := logger.New(os.Stdout, cfg.LogLevel)
```